users/views.py

```diff
@@ -1,3 +1,57 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import re
+import requests
 
-# Create your views here.
+from twilio.rest  import Client
+from django.views import View
+from django.http  import JsonResponse
+from random       import randint
+from users.models import User, PhoneCheck
+from my_settings  import ACCOUNT_SID, AUTH_TOKEN
+
+
+class SendSmSView(View):
+    def post(self, request):
+        data         = json.loads(request.body)
+        input_mobile = data['mobile']
+        try:
+            if re.match(r"(010)\d{4}\d{4}",input_mobile):
+                to_mobile = '+82'+ input_mobile[1:]
+            else:
+                return JsonResponse({'message':'INVALID_NUMBER'}, status=400)
+            
+            check_number = randint(1000,9999)
+            mobile_db    = PhoneCheck.objects.create(check_id=input_mobile, check_number=check_number)
+
+            account_sid  = ACCOUNT_SID
+            auth_token   = AUTH_TOKEN
+            client       = Client(account_sid, auth_token)
```

Comment:
굳이 또 담지 않고 바로 전달해도 될것 같은데요~!
```suggestion
            client       = Client( ACCOUNT_SID, AUTH_TOKEN)
```

Reply:
수정 진행 하도록 하겠습니다. 감사합니다.